src/constants/relay.const.ts

```diff
@@ -0,0 +1 @@
+export const ESTIMATED_GAS_CORRECTION_FACTOR = '1';
```

Comment:
really shouldn't be a string if it is a number/BN.

Reply:
its a bigNumber, I had it like that but Im not sure now. Why cannot be a string?

Reply:
@franciscotobar I think Juraj was proposing to specify the type there as BigNumber directly. @jurajpiar was that what you meant?